src/main/java/frc/robot/subsystems/gripper/GripperConstants.java

```diff
@@ -16,17 +23,28 @@ public class GripperConstants {
     public static final Measure<Distance> GRIPPER_POSITION_Y = Units.Meters.of(0);
     public static final Measure<Distance> GRIPPER_POSITION_z = Units.Meters.of(0.6461);
     public static final double ANGLE_MOTOR_GEAR_RATIO = 58.5;
+    public static final InvertedValue ANGLE_INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    public static final boolean ROLLER_INVERTED_VALUE = true;
+    public static final int CURRENT_LIMIT = 40;
+    public static final LoggedTunableNumber ABSOLUTE_ENCODER_OFFSET =
+            new LoggedTunableNumber("Gripper/EncoderOffset");
 
     public static final LoggedTunableNumber KP = new LoggedTunableNumber("Gripper/kP");
     public static final LoggedTunableNumber KI = new LoggedTunableNumber("Gripper/kI");
     public static final LoggedTunableNumber KD = new LoggedTunableNumber("Gripper/kD");
+    public static final LoggedTunableNumber KV = new LoggedTunableNumber("Gripper/kD");
+    public static final LoggedTunableNumber KA = new LoggedTunableNumber("Gripper/kD");
+    public static final LoggedTunableNumber KG = new LoggedTunableNumber("Gripper/kD");
```

Comment:
If you're using kG you need to configure `Gravitytype`.

Reply:
eac9c751b88757ccb73f0fd3260eeba16616adbe